api/db/migrate/20190606151443_create_position_data_for_ads.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePositionDataForAds < ActiveRecord::Migration[5.2]
+  def change
+    create_table :position_data_for_ads do |t|
```

Comment:
Did we ever come to a conclusion on whether or not we should be on one singular position table or multiple position tables?

Reply:
@reidka mentioned in the meeting yesterday to separate position into multiple tables.